docs/adr/0002-delivery-channel-creation-and-integration.md

```diff
@@ -0,0 +1,102 @@
+# New delivery channel creation and integration
+
+## Table of definitions
+
+| Word     | Definition                                                                                                                            |
+|----------|---------------------------------------------------------------------------------------------------------------------------------------|
+| Consumer | A service that uses the Marain.UserNotifications service                                                                              |
+| User     | The individual who is using an application that consumes the Marain.UserNotifications service                                         |
+
+## Status
+
+Proposed
+
+## Context
+
+Delivering notifications through different communication types (email, sms, web push notifications etc) is necessary in an extendable notifications service. Organizations should be able to choose which delivery channels (twilio, sendgrid, airship etc) they want to use for the associated communication type(s) as these said platforms could have different features or costs. Sometimes using different communication types through multiple delivery channels could save costs rather than having multiple communication types through a single delivery channel.
+
```

Comment:
I find this last sentence a bit confusing. For me "using different communication types through multiple delivery channels" could mean either of:
- Having more than one delivery channel that supports a communication type (e.g. SMS) and using all of them to send that type of notification.
- Having more than one delivery channel that supports a communication type (e.g. SMS), but selecting a specific one to send that type of communication.

I'm assuming you mean the latter..

Reply:
Agreed, the sentence is confusing. I've changed the sentence to be a little clearer.